shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -18,107 +18,55 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol FlutterPluginRegistrar;
 @protocol FlutterPluginRegistry;
 
-/**
- * A plugin registration callback.
- *
- * Used for registering plugins with additional instances of
- * `FlutterPluginRegistry`.
- *
- * @param registry The registry to register plugins with.
- */
-typedef void (*FlutterPluginRegistrantCallback)(NSObject<FlutterPluginRegistry>* registry);
-
-/**
- * Implemented by the iOS part of a Flutter plugin.
- *
- * Defines a set of optional callback methods and a method to set up the plugin
- * and register it to be called by other application components.
- */
-@protocol FlutterPlugin <NSObject>
-@required
-/**
- * Registers this plugin using the context information and callback registration
- * methods exposed by the given registrar.
- *
- * The registrar is obtained from a `FlutterPluginRegistry` which keeps track of
- * the identity of registered plugins and provides basic support for cross-plugin
- * coordination.
- *
- * The caller of this method, a plugin registrant, is usually autogenerated by
- * Flutter tooling based on declared plugin dependencies. The generated registrant
- * asks the registry for a registrar for each plugin, and calls this method to
- * allow the plugin to initialize itself and register callbacks with application
- * objects available through the registrar protocol.
- *
- * @param registrar A helper providing application context and methods for
- *     registering callbacks.
- */
-+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar;
-@optional
-/**
- * Set a callback for registering plugins to an additional `FlutterPluginRegistry`,
- * including headless `FlutterEngine` instances.
- *
- * This method is typically called from within an application's `AppDelegate` at
- * startup to allow for plugins which create additional `FlutterEngine` instances
- * to register the application's plugins.
- *
- * @param callback A callback for registering some set of plugins with a
- *     `FlutterPluginRegistry`.
+#pragma mark -
+/***************************************************************************************************
```

Comment:
Do these long lines do anything special with doxygen or something?

Reply:
Nope, they are ignored by doxygen.  It's for the benefit of someone scrolling through the file to see breaks between the top level objects.